packages/design-system/src/styles/components/_Dropdown.scss

```diff
@@ -58,16 +66,48 @@
   .ds-c-field--inverse + & {
     color: var(--color-base);
   }
+
+  // When dropdown is open (not `hidden`), add border radius to bottom of menu
+  &:not([hidden]) {
+    // We don't have a good way to create alpha versions of colors using our current hex value tokens
+    // Sass has color functions, but they can't be used in CSS custom properties because CSS custom properties are evaluated at runtime
+    // Hex values can't be used in native CSS color functions like `hsla()` or `rgba()`
+    // Creating a temporary variable until we discuss a better solution
+    --shadow-color: 0deg 0% 50%;
+    animation: fade-in var(--animation-speed-1) var(--animation-ease-in-out-expo);
+    border-radius: 0 0 var(--dropdown-menu__border-radius--open)
+      var(--dropdown-menu__border-radius--open);
+    box-shadow: 0px 0.2px 0.2px hsl(var(--shadow-color) / 0.43),
+      0px 0.7px 0.7px -0.5px hsl(var(--shadow-color) / 0.39),
+      0px 1.4px 1.3px -1.1px hsl(var(--shadow-color) / 0.35),
+      -0.1px 2.6px 2.4px -1.6px hsl(var(--shadow-color) / 0.32),
+      -0.1px 4.6px 4.3px -2.1px hsl(var(--shadow-color) / 0.28),
+      -0.2px 7.8px 7.3px -2.7px hsl(var(--shadow-color) / 0.25),
+      -0.3px 12.6px 11.8px -3.2px hsl(var(--shadow-color) / 0.21),
+      -0.4px 19.1px 17.9px -3.7px hsl(var(--shadow-color) / 0.18);
+  }
 }
 
 .ds-c-dropdown__menu {
   @include mixins.ds-unstyled-list;
   gap: 0;
 }
 
+// Animation keyframes for dropdown menu
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+// Base styles for grouped and individual dropdown elements
 .ds-c-dropdown__item,
 .ds-c-dropdown__item-group {
   border-block-end: var(--dropdown-item__border-width) solid var(--dropdown-item__border-color);
+  cursor: default;
```

Comment:
I don't know if we wanted the cursor to look like an arrow when hovering over this element, but I figured we'd want it to behave as close to a native select as possible, and select options use this cursor style.